OrbitGl/GraphTrack.cpp

```diff
@@ -32,15 +33,26 @@ void GraphTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
   }
 
   float track_z = GlCanvas::kZValueTrack;
-  float text_z = GlCanvas::kZValueText;
 
   Box box(pos_, Vec2(size_[0], -size_[1]), track_z);
 
   // Draw label with graph value at current mouse position.
-  const Color kLabelTextColor(255, 255, 255, 255);
+  const Color kWhiteColor(255, 255, 255, 255);
+  const Color kBlackColor(0, 0, 0, 255);
   double graph_value = GetValueAtTime(time_graph_->GetCurrentMouseTimeNs());
-  canvas->GetTextRenderer().AddText(std::to_string(graph_value).c_str(), canvas->GetMouseX(), y0,
-                                    text_z, kLabelTextColor);
+
+  Vec2 arrow_box_size(50., layout.GetTextBoxHeight());
+  auto rightmost_x_position = canvas->GetWorldTopLeftX() + canvas->GetWorldWidth() -
+                              layout.GetRightMargin() - layout.GetSliderWidth() - arrow_box_size[0];
+  Vec2 arrow_box_position(std::min(canvas->GetMouseX(), rightmost_x_position),
+                          y0 - static_cast<float>((max_ - graph_value) * size_[1] / value_range_) -
+                              arrow_box_size[1] / 2);
```

Comment:
Vec2 is float, right? Then 2 should be 2.f here.

Reply:
Done!